Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
 import './App.css';
+import NumberButton from "./components/ButtonComponents/NumberButton"
+import ActionButton from "./components/ButtonComponents/ActionButton"
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay"
+import MathButton from "./components/ButtonComponents/MathButton"
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="App">
```

Comment:
Good job rendering all your child components in App.js! Your calculator looks great.